Board.java

```diff
@@ -0,0 +1,162 @@
+import java.sql.SQLOutput;
+
+public class Board {
+
+    enum Square {
```

Comment:
Why defining a limited number of colors and limit yourself to two players?! SUCH A KID